README.md

```diff
@@ -298,3 +298,8 @@ $ git pull upstream master
 $ git push origin master
 $ git branch -d mybranch
 ```
+
+## Wiki: ##
+----------
+All wikis have been moved to their own repository [e-mission-docs](https://github.com/e-mission/e-mission-docs). E-mission phone specific wikis can be found here:
```

Comment:
Should this be at the end or at the beginning?
Also, maybe instead of "all wikis", say "additional documentation", since the new repository is not really a wiki any more :)